src/adapter.js

```diff
@@ -0,0 +1,24 @@
+function Adapter(code) {
```

Comment:
yes, this replaces `src/adpaters/adapter.js`.  I haven't removed the old file yet because I haven't finished converting all the adapters to modules yet, so the non-converted adapters still look for this file.  As soon as I'm done converting the adapters to modules I'll make sure this is moved properly in git.